content/blog/2022/05/verifying-image-signatures.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Secure Supply Chain: Verifying image signatures"
+authors:
+- Raul Cabello Martin
+date: 2022-05-02
+---
+
+After these last releases Kubewarden now has support for verifying the integrity and authenticity of artefacts within
+Kubewarden using the Sigstore project. In this post, we shall focus on verifying container image signatures using the
+new [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy
+
+If you don't know how Sigstore works, we recommend reading our
+previous [post](https://www.kubewarden.io/blog/2022/04/securing-kubewarden-policies/)
+
+## Verify Image Signatures Policy
+
+This policy validates Pods by checking their container images for signatures (that is, containers, init containers and 
+ephemera containers in the pod). If all signature validations pass or there is no container image that matches the image
+name query, the Pod will be accepted.
+
+Once the image has been verified with a signature, one needs to ensure that the specific image tag from that signature 
+is the one instantiated. This is achieved by the policy by mutating the Pod request and adding that validated image digest. 
```

Comment:
Just a simple suggestion to remove the `by ... by`. If you think this is not necessary. Just mark the comment as resolved. ;)

```suggestion
is the one instantiated. The policy achieves this by mutating the pod request and adding that validated image digest
```

Reply:
it has been changed [here](https://github.com/kubewarden/kubewarden.io/pull/100/commits/a0ae28cf227cfff1a42cfaa4f8b39b3a76bf90b4)